src/pages/trainingVideos.tsx

```diff
@@ -26,19 +27,36 @@ export const TrainingVideosPage = () => (
       </div>
       <div className="row py-3">
         <div className="col-12 col-lg-4 mb-4">
-          <MediaCard title="TradeTrust Overview" youtubeEmbedCode="3-ZRuPCa2k4">
+          <MediaCard title="TradeTrust Overview" youtubeEmbedCode="NcR1M9NJ-PE">
             <p>
               This non-technical session helps provide a foundational and critical understanding of TradeTrust as a
               digital utility as well as the mental framing necessary as a pre-requisite for subsequent webinars.
             </p>
+            <div className="py-2">
+              <AttachmentLink
+                className="mb-3"
+                filename="TradeTrust Tech Webinar 1 - Overview.pdf"
+                path="/static/images/webinar/tradetrust-tech-webinar-1-overview.pdf"
```

Comment:
Let's hear what the other's have to say about storing big blobs in the folder first. I'm not certain it will be a good idea.